### DISCUSSION TITLE
How long will my domain on pending verification?

### DISCUSSION THREAD

**Discussion by a870588** [vercel/community#2014] [Help] [answered]
### Summary

<img width="1436" alt="Screen Shot 2023-04-08 at 10 15 21 PM" src="https://user-images.githubusercontent.com/126646682/230726101-5f252609-1200-4066-a90c-73952c66f670.png">

Hello, guys

Recently, I just transferred my project and domain from my account A to account B (in the picture), and I deleted my account A after the transfer. I set up the domain on my account B, and TXT on my domain provider as the instruction suggested it. But, after I have waiting for 24 hours, it's still on pending verification. 

I searched on the information, one old post from last year said: "This happens when previous account was deleted, and I need to wait for **4 DAYS** (some other said 2 days) because Vercel preserves my old account in case I want to reverse the deletion."

My question is:

1. How long may I have to wait for my domain to be verified and configured?
2. Are there any way may I expedite the process?

I have mailed the support team and their auto-generated mail mentioned that "ticket will be prioritized based on my account plan", which mean I may need to wait for a long time. Any suggestion and help is greatly appreciated, thanks.

**Comment by amyegan** [comment, marked as answer]
Hello @a870588. I don't see the record when checking global DNS propagation (https://www.whatsmydns.net/#TXT/_vercel.waynehsu.site). Please double check that your TXT is configured correctly for `_vercel` and let us know if you're still having trouble with this 🙏
